data/transition-sites/gds_alphagov.yml

```diff
@@ -0,0 +1,9 @@
+---
+site: gds_alphagov
+whitehall_slug: government-digital-service
+homepage: https://www.gov.uk/government/organisations/government-digital-service
+tna_timestamp: 20201010101010 # Stub timestamp - site not in TNA
```

Comment:
"site not in TNA", but commit message implies it _is_ in the national archives?

Reply:
Oops, I missed a keyword of "not". It's intentional to flag this as not in the national archives as the data in the national archives is only an unhelpful redirect.